external/mbd/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ set(ENABLE_C_API OFF)
 if(WITH_MPI)
   get_target_property(SCALAPACK_LIBRARIES Scalapack::Scalapack INTERFACE_LINK_LIBRARIES)
 endif()
+if(WITH_ELSI)
+  # FIXME: Hacky CMake magic
+  #
+  # libmbd will link against ELSI using the ELSI_LIBRARIES variable and ignore the
+  # elsi::elsi target, but that doesn't mean we can't trick it via CMake variable
+  # expansion to still use the correct target:
+  set(ELSI_LIBRARIES elsi::elsi)
```

Comment:
@aradi Hope this is okay for you.

Reply:
@awvwgk Could you test, whether libmbd/libmbd#31 would solve the problem? Then probably won't need this horrible hack. :wink:

Reply:
That would work even better.